.github/workflows/phpbench.yml

```diff
@@ -51,4 +51,4 @@ jobs:
         uses: "ramsey/composer-install@v3"
 
       - name: "Run PHPBench"
-        run: "vendor/bin/phpbench run --report=default"
+        run: "ENABLE_NATIVE_LAZY_OBJECTS=0 vendor/bin/phpbench run --report=default"
```

Comment:
This line does not work with native lazy objects: https://github.com/doctrine/orm/blob/92e2f6db831be44bc041fdfbc49402a063ec33cc/tests/Performance/LazyLoading/ProxyInitializationTimeBench.php#L39, so I'm disabling them, postponing the decision of what to do with this benchmark to something else than a merge up PR.